package/MDAnalysis/analysis/helanal.py

```diff
@@ -131,6 +135,12 @@
 import logging
 logger = logging.getLogger("MDAnalysis.analysis.helanal")
 
+warnings.warn("This module is deprecated as of MDAnalysis version 1.0. "
```

Comment:
We'll probably want some test for this? (not sure how strict we are being with 1.0.1 tests, but if it's getting released we should try to get coverage down?)

Reply:
Actually the tests are failing here, so we it'll need fixing anyways.